lib/foreman_leapp/engine.rb

```diff
@@ -81,11 +81,16 @@ class Engine < ::Rails::Engine
       Foreman::Gettext::Support.add_text_domain locale_domain, locale_dir
     end
 
-    initializer 'ForemanLeapp.require_dynflow',
+    initializer 'foreman_leapp.require_dynflow',
                 before: 'foreman_tasks.initialize_dynflow',
                 after: 'foreman_remote_execution.require_dynflow' do |_app|
       ForemanTasks.dynflow.require!
       ForemanTasks.dynflow.config.eager_load_paths << File.join(ForemanLeapp::Engine.root, 'app/lib/actions')
     end
+
+    initializer('foreman_leapp.extend_remote_execution',
+                after: 'foreman_leapp.require_foreman_remote_execution') do |_app|
+      RemoteExecutionHelper.prepend ForemanLeapp::RemoteExecutionHelperExtension
```

Comment:
Why not do this where all the other extensions happen https://github.com/oamg/foreman_leapp/pull/29/files#diff-8d68f56fa2177f1d565d390c12f94e3eR43-L45 ?

Reply:
I tried it but it didn't work. What I tried:
```
config.to_prepare do
  RemoteExecutionHelper.prepend ForemanLeapp::RemoteExecutionHelperExtension
end
```
 `RemoteExecutionHelperExtension::job_invocation_task_buttons` was never called. This is the only way that worked for me.

Reply:
Huh, interesting. Nevermind then